rust/openvasd/src/config.rs

```diff
@@ -597,8 +598,9 @@ impl Config {
         if let Some(path) = cmds.get_one::<PathBuf>("tls-client-certs") {
             config.tls.client_certs = Some(path.clone());
         }
-        if let Some(enable) = cmds.get_one::<bool>("enable-get-scans") {
-            config.endpoints.enable_get_scans = *enable;
+        if let Some(enable) = cmds.get_one::<String>("enable-get-scans") {
+            config.endpoints.enable_get_scans =
+                matches!(enable.to_lowercase().as_str(), "true" | "t" | "1");
```

Comment:
can you explain why we need to have multiple ways to express 'true|false' within the cmd line?